### PR TITLE
ROX-19093: Replace success state with generated delivered in rep history

### PR DESCRIPTION
**Pull request by ksurabhi91**
This PR removes conversion of generated/delivered to success in api v2 config. 

## Testing Performed
Manually verified runstate field in history response
Screenshot 2023-09-06 at 10 41 36 AM

### CODE REVIEW

**Review by charmik-redhat — COMMENTED** [review]
Changes look good. Fix PR title and add testing steps/screenshots to the PR description. Manual testing should be fine.

**Comment by sachaudh** [comment]
My PR is approved. I'll wait until CI is good and wait until this is approved so we can merge them together

**Review by charmik-redhat — APPROVED** [review]
Good to ship

**Pull request merged**
